pennylane/interfaces/batch/__init__.py

```diff
@@ -74,7 +72,7 @@ def wrapper(tapes, **kwargs):
             # No caching. Simply execute the execution function
             # and return the results.
             res = fn(tapes, **kwargs)
-            return res, [] if return_tuple else res
+            return (res, []) if return_tuple else res

```

Comment:
This is a bug I noticed while fixing this PR

Reply:
I thought so as well 😆 

Unfortunately, Python interprets it as follows if you don't explicitly include the tuple parenthesis:

```python
return res, ([] if return_tuple) else res)
```

Reply:
:astonished: That's quite a trap! Nice catch. :slightly_smiling_face: